case-study/sqls/action-statistical-characteristics/sql

```diff
@@ -0,0 +1,62 @@
+SELECT repo_id,
+    anyHeavy(repo_name) AS repo_name,
+    floor(avg(resolve_period)) AS resolve_period_avg,
+    floor(avg(respond_period)) AS respond_period_avg,
+    median(resolve_period) AS resolve_period_median,
+    median(respond_period) AS respond_period_median,
+    count(1) AS count
+FROM (
+        SELECT repo_id,
+            repo_name,
+            issue_number,
+            close_at - open_at As resolve_period,
+            interact_at - open_at AS respond_period
+        FROM (
+                SELECT oi.repo_id As repo_id,
+                    oi.repo_name AS repo_name,
+                    oi.issue_number AS issue_number,
+                    oi.time AS open_at,
+                    ci.time AS close_at,
+                    ii.time As interact_at
+                FROM (
+                        SELECT repo_id,
+                            anyHeavy(repo_name) AS repo_name,
+                            issue_number,
+                            min(created_at) AS time
+                        FROM github_log.year2020
```

Comment:
You need to replace the table name with `{{table}}`.

Reply:
Great advice~